src/cli/index.ts

```diff
@@ -49,7 +49,11 @@ async function sign(artifactPath: string) {
 
 async function signDSSE(artifactPath: string, payloadType: string) {
   const buffer = fs.readFileSync(artifactPath);
-  const bundle = await dsse.sign(buffer, payloadType, signOptions);
+  const bundle = await sigstore.signAttestation(
+    buffer,
+    payloadType,
```

Comment:
Should the payload type default to `application/vnd.in-toto+json`?

https://github.com/in-toto/attestation/blob/main/spec/README.md#envelope

Reply:
Yes, great idea!